decentraland/redux/actions.js

```diff
@@ -63,3 +64,10 @@ export function removeMyAssetFromList(myAsset) {
 export function setMyAssetsList(myAssets) {
   return { type: SET_MY_ASSETS_LIST, myAssets };
 }
+
+export function setActionIdForMyAsset(myAssetId, actionId) {
```

Comment:
We could even pass them in in the format that they're reformatted to below for consistency's sake

Reply:
Didn't got it

Reply:
`export function setActionIdForMyAsset(paramObject) {`

Where paramObject (note that we'd want to use a better name) is like this:
```js
{
  myAssetId: myAssetId,
  actionId: actionId
}
```

Reply:
(Note that this doesn't gate merging)

Reply:
Ah, ok. Got it.